### PR TITLE
feat(server-cloudevents): document type in compliance with CloudEvents spec

### PR DESCRIPTION
**Pull request by JoergSiebahn**

### CODE REVIEW

File: sda-commons-server-cloudevents/asyncapi.yaml [hunk 1]
```diff
@@ -251,7 +253,9 @@ components:
           type: "string"
           default: "PARTNER_CREATED"
```
Comment (christopher-cudennec) [inline sda-commons-server-cloudevents/asyncapi.yaml:254]: Should also match the examples.

File: sda-commons-server-cloudevents/asyncapi.yaml [hunk 2]
```diff
@@ -251,7 +253,9 @@ components:
...
           examples:
-          - "CONSENT_VERSION_CREATED"
+          - "com.sdase.relocation.service.customer.moved"
+          - "com.sdase.consent.service.consent.agreed"
+          - "com.sdase.offer.service.contract.offered"
           minLength: 1
           description: "`PARTNER_CREATED`"
```
Comment (christopher-cudennec) [inline sda-commons-server-cloudevents/asyncapi.yaml:260]: Should also match the examples.

Comment (JoergSiebahn) [reply sda-commons-server-cloudevents/asyncapi.yaml]: Good catch. [fixed](https://github.com/SDA-SE/sda-dropwizard-commons/pull/1783/commits/a5123945bf6e92526a12ff4874efb41739d68bfb)

File: sda-commons-server-cloudevents/asyncapi.yaml [hunk 3]
```diff
@@ -107,19 +107,21 @@ components:
           type: "string"
           default: "CONTRACT_CREATED"
```
Comment (christopher-cudennec) [inline sda-commons-server-cloudevents/asyncapi.yaml:108]: Please also apply reverse DNS name. Thanks! 🌻

**Review by christopher-cudennec — APPROVED** [review]
🏇

**Pull request merged**